resources/assets/components/utilities/SiteFooter/SiteFooter.js

```diff
@@ -35,7 +35,9 @@ const SiteFooter = () => {
                 title="dosomething on Tik Tok"
                 className="hover:text-white"
                 style={{ padding: 0 }}
-                onClick={handleFooterTracking('tiktok')}
+                onClick={event =>
+                  handleFooterTracking('tik_tok', event.target.href)
```

Comment:
I think we can keep this one as `tiktok` since the title is a single word?
```suggestion
                  handleFooterTracking('tiktok', event.target.href)
```